internal/model/toc.go

```diff
@@ -232,6 +238,36 @@ func matchesFilter(acceptedValues []string, value string) bool {
 	return slices.Contains(acceptedValues, utils.SanitizeName(value))
 }
 
+func excludeBySimpleContentSearch(searchQuery string) func(e *IndexEntry) bool {
```

Comment:
looks like this is the inlined `func (e *IndexEntry) MatchesSearchText()`, but the original is not removed. why?